### PR TITLE
Styling fixes

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: paper-slider.html [hunk 1]
```diff
@@ -199,7 +199,7 @@
         width: calc(100% - 20px);
         height: calc(100% - 20px);
         background-color: var(--paper-slider-knob-color, --google-blue-700);
-        border: 1px solid var(--paper-slider-knob-color, --google-blue-700);
+        border: 2px solid var(--paper-slider-knob-color, --google-blue-700);
```
Comment (keanulee) [inline paper-slider.html]: Don't transition border width anymore

**Comment by blasten** [comment]
screen shot 2016-01-07 at 5 11 52 pm

The knob in the disabled state is clipped in a weird way. It seems like a bug in Chrome because it renders fine in Safari and Firefox.  `will-change: transform;` fixes it.

**Comment by keanulee** [comment]
Whoa, putting `#sliderKnobInner` on its own layer causes an issue with the pin with Chrome/Firefox/Safari. This is pretty unexpected, and I'll look into it.
screen shot 2016-01-07 at 5 25 06 pm 2

**Comment by keanulee** [comment]
Never mind, only had to deal with the disabled knob state (which is not `scale(0.75)` instead of smaller layout size). Switching that to 3D transform seems to fix the issue.

**Comment by blasten** [comment]
great job @keanulee

**Pull request merged**